app/components/work_package_relations_tab/index_component.rb

```diff
@@ -9,16 +15,22 @@ class WorkPackageRelationsTab::IndexComponent < ApplicationComponent
   include Turbo::FramesHelper
   include OpTurbo::Streamable
 
-  attr_reader :work_package, :relations, :children, :directionally_aware_grouped_relations, :scroll_to_id
+  attr_reader :work_package, :relations, :children, :directionally_aware_grouped_relations, :relation_to_scroll_to
 
-  def initialize(work_package:, relations:, children:, scroll_to_id: nil)
+  # Initialize the component with required data
+  #
+  # @param work_package [WorkPackage] The work package whose relations are being displayed
+  # @param relations [Array<Relation>] The relations associated with this work package
+  # @param children [Array<WorkPackage>] Child work packages
+  # @param relation_to_scroll_to [Relation, WorkPackage, nil] Optional relation or child to scroll to when rendering
+  def initialize(work_package:, relations:, children:, relation_to_scroll_to: nil)
```

Comment:
Added this doc to emphasize that `relation_to_scroll_to` is either a `Relation` or a `WorkPackage` (child relation).